include/string.h

```diff
@@ -91,10 +91,10 @@ int strerror_r (int, char *, size_t);
 char *stpcpy(char *__restrict, const char *__restrict);
 // Copies at most n characters from the string pointed to by src, including the terminating null byte ('\0'),
 // to the array pointed to by dest.
-char *stpncpy(char *restrict d : itype(restrict _Nt_array_ptr<char>) count(n),
+char *stpncpy(char *restrict d : itype(restrict _Array_ptr<char>) count(n),
```

Comment:
I would recommend parameter `d` be annotated as a null-terminated pointer, since `stpncpy` takes a string, and use `count(n-1)` instead of `count(n)`.

Reply:
In stpncpy, we do a void cast for wd pointer and assign it to d. Since we do not have the void cast for _Nt_array_ptr type I used _Array_ptr. I am not sure if I've already tried the cast of _Nt_array_ptr<char> cast instead of void cast. I will check it again.